packaging/ubuntu-14.04/snapd.links

```diff
@@ -1,2 +1,4 @@
 ../../usr/lib/snapd/snap-device-helper  /lib/udev/snappy-app-dev
 usr/lib/snapd/snapctl usr/bin/snapctl
+usr/share/dbus-1/session.d/snapd.session-services.conf etc/dbus-1/session.d/snapd.session-services.conf
+usr/share/dbus-1/system.d/snapd.system-services.conf etc/dbus-1/system.d/snapd.system-services.conf
```

Comment:
Should we not ship this on 14.04?

Reply:
And a related question, if we choose not to ship it on 14.04 - should we make it so that on re-exec, this feature is not supported?

Perhaps we should add a function to the each feature, that checks if this is supported by the running system? Not sure